src/components/StaffList/index.jsx

```diff
@@ -233,7 +235,8 @@ class StaffList extends React.Component {
           </Droppable>
         </DragDropContext>
         {/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */}
-        <label className="col-12" id="label-staff-search" htmlFor="staff-search" onKeyDown={this.handleAutosuggestEnterEvent}>Search or add new instructor
+        <label className={suggestionVisible ? 'w-100 suggestions-visible' : 'w-100'} id="label-staff-search" htmlFor="staff-search" onKeyDown={this.handleAutosuggestEnterEvent}>
```

Comment:
There's a library called `classNames` that is already used in publisher-frontend that is a cleaner way to handle this ternary operator.

Reply:
Don't need to write the class dynamically anymore, but good to know!